sdk/python/kfp/aws.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def use_aws_secret(secret_name='aws-secret', aws_access_key_id_name='AWS_ACCESS_KEY_ID', aws_secret_access_key_name='AWS_SECRET_ACCESS_KEY'):
+    """An operator that configures the container to use AWS credentials.
+
+        AWS doesn't create secret along with kubeflow deployment and it requires users
+        to manually create credential secret with proper permissions.
+        ---
+        apiVersion: v1
+        kind: Secret
+        metadata:
+          name: aws-secret
+        type: Opaque
+        data:
+          AWS_ACCESS_KEY_ID: YOUR_AWS_ACCESS_KEY_ID
+          AWS_SECRET_ACCESS_KEY: YOUR_AWS_SECRET_ACCESS_KEY
+
+        We would suggest users to attach minimum policis or roles to get AWS APIs access.
```

Comment:
```suggestion
        We would suggest users to attach minimum policies or roles to get AWS APIs access.
```

This line is a bit unclear. What exactly do we want from users?

Reply:
Instead of creating a role with lots of permissions, I think it's better to ask users to attach minimum permission based on their needs. 

This line is not helpful in the code, I will remove it.